media-libs/libfdk_aac/libfdk_aac-0.1.5.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="Fraunhofer AAC codec library"
+DESCRIPTION='The Fraunhofer FDK AAC Codec Library for Android ("FDK AAC Codec") \
+	is software that implements the MPEG Advanced Audio Coding ("AAC") \
+	encoding and decoding scheme for digital audio.'
+HOMEPAGE="https://sourceforge.net/projects/opencore-amr"
+COPYRIGHT="1995-2012 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e.V."
+LICENSE="Fraunhofer FDK AAC Codec"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/opencore-amr/fdk-aac/fdk-aac-$portVersion.tar.gz"
+CHECKSUM_SHA256="2164592a67b467e5b20fdcdaf5bd4c50685199067391c6fcad4fa5521c9b4dd7"
+SOURCE_DIR="fdk-aac-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+libVersion="1.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	libfdk_aac$secondaryArchSuffix = $portVersion
+	lib:libfdk_aac$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libfdk_aac${secondaryArchSuffix}_devel = $portVersion
+	devel:libfdk_aac$secondaryArchSuffix = $libVersion
+	"
+REQUIRES_devel="
+	haiku${secondaryArchSuffix}_devel
+	libfdk_aac$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:awk
+	cmd:g++$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+defineDebugInfoPackage libfdk_aac$secondaryArchSuffix \
+	$libDir/libfdk-aac.so.$libVersion
+
+BUILD()
+{
+	runConfigure ./configure --enable-shared
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	install -d -m 755 "$docDir"
+	install -t "$docDir" -m 644 ChangeLog
+
+	rm -rf "$libDir"/*.la
```

Comment:
No need for recursion here ;-)
```diff
-	rm -rf "$libDir"/*.la
+	rm -f "$libDir"/*.la
```

BTW, everything else looks ok to me too. Great job, @alaviss, once more ;-)
I'll try to build this recipe on x86_gcc2 for the x86 secondary arch.
So we might even be able to mark x86 as tested in SECONDARY_ARCHITECTURES.